evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1165,9 +1173,14 @@ def test_automl_time_series_classification_threshold(
     automl.search()
     assert isinstance(automl.data_splitter, TimeSeriesSplit)
     if objective == "Log Loss Binary":
-        mock_optimize_threshold.assert_not_called()
-        assert automl.best_pipeline.threshold is None
-        mock_split_data.assert_not_called()
+        if not optimize:
+            mock_optimize_threshold.assert_not_called()
+            assert automl.best_pipeline.threshold is None
+            mock_split_data.assert_not_called()
+        else:
+            mock_optimize_threshold.assert_called()
+            assert automl.best_pipeline.threshold == 0.62
+            mock_split_data.assert_called()
     elif optimize and objective == "F1":
         mock_optimize_threshold.assert_called()
         assert automl.best_pipeline.threshold == 0.62
```

Comment:
So should this move into the above else branch and be a split on mock_split_data being called or not?